manifests/charts/istiod-remote/values.yaml

```diff
@@ -315,6 +315,8 @@ defaults:
     proxy_init:
       # Base name for the proxy_init container, used to configure iptables.
       image: proxyv2
+      # Allows override if detection incorrectly assumes rules are already applied. Consider removing after several releases with no reported issues.
+      force_apply_iptables: false
```

Comment:
should be `forceApplyIptable`?

Reply:
oh yes sorry, wrong convention :p
Updated